server/db/tables/products.js

```diff
@@ -24,7 +24,7 @@ const getProductsByCategory = categoryId => {
 /* Returns all products by user id */
 const getProductsByUser = userId => {
   return pool.query({
-    text: `select * from products where user_id = ${userId}`
```

Comment:
Any of these querries that are using template literals need to be refactored into one of two patterns, otherwise they are open to SQL injection attacks:
` select * from products where user_id = "${userId}"` Surround template literal with string quotes.
` 'select * from products where user_id = $1', [userId]` Use the replacement pattern in querry where the variable is in an array as the second parameter of pool.query()